docs/modules/ROOT/pages/clc-viridian.adoc

```diff
@@ -62,3 +66,68 @@ Usage:
 ----
 clc viridian list-clusters [flags]
 ----
+
+== clc viridian list-custom-classes
+
+Lists all custom classes in the Viridian Cluster.
+
+Make sure you login before running this command.
+
+Usage:
+
+[source,bash]
+----
+clc viridian list-custom-classes <cluster-name/cluster-id> [flags]
+----
+
+== clc viridian upload-custom-class
+
+Uploads a custom class to the Viridian Cluster.
+
+Make sure you login before running this command.
+
+Usage:
+
+[source,bash]
+----
+clc viridian upload-custom-class <cluster-name/cluster-id> <fileName/artifact-id> [flags]
+----
+
+== clc viridian download-custom-class
+
+Downloads a custom class from the Viridian Cluster.
+
+Make sure you login before running this command.
+
+Usage:
+
+[source,bash]
+----
+clc viridian download-custom-class <cluster-name/cluster-id> <fileName/artifact-id> [flags]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`--output`
```

Comment:
Let's make sure the flags in the docs correspond to the implementation. Same comment for other commands.